src/envo/envo-edit.owl

```diff
@@ -4115,6 +4127,23 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.ob
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001307> "Definition modified from 'surrounded by'.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001307> "partially_surrounded_by")
 
+# Object Property: <http://purl.obolibrary.org/obo/ENVO_02000139> (used for transport of)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000139> "A relation between a material entity and a material where the function of the material entity is realized as a process that results in the movement of the material.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000232> <http://purl.obolibrary.org/obo/ENVO_02000139> "This may be later moved to RO")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000139> "cjm"^^xsd:string)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000139> "used for transport of"@en)
+ObjectPropertyDomain(<http://purl.obolibrary.org/obo/ENVO_02000139> <http://purl.obolibrary.org/obo/BFO_0000040>)
+ObjectPropertyRange(<http://purl.obolibrary.org/obo/ENVO_02000139> <http://purl.obolibrary.org/obo/BFO_0000040>)
+
+# Object Property: <http://purl.obolibrary.org/obo/ENVO_02000145> (used for processing of)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000145> "A relation between a two material entities where the function of the first is realized as a process that has the second as input.")
```

Comment:
As above. Also, as above, the phrasing of the definition is unclear - how can a function be realised as a processes?